src/Ontology/Oep4.h

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Address.h"
+#include "Asset.h"
+#include "ParamsBuilder.h"
+#include "Transaction.h"
+#include "../Data.h"
+
+namespace TW::Ontology {
+
+class Oep4 {
+private:
+    static const uint8_t version = 0x00;
```

Comment:
we can use constexpr instead of const here

Reply:
use constexpr instead